monkey/tests/unit_tests/monkey_island/cc/services/telemetry/processing/credentials/test_mimikatz_processing.py

```diff
@@ -0,0 +1,92 @@
+from copy import deepcopy
+from datetime import datetime
+
+import dpath.util
+import mongoengine
+import pytest
+
+from common.config_value_paths import (
+    LM_HASH_LIST_PATH,
+    NTLM_HASH_LIST_PATH,
+    PASSWORD_LIST_PATH,
+    USER_LIST_PATH,
+)
+from monkey_island.cc.services.config import ConfigService
+from monkey_island.cc.services.telemetry.processing.credentials.credentials_parser import (
+    parse_credentials,
+)
+
+MIMIKATZ_TELEM_TEMPLATE = {
```

Comment:
Can we remove the references to mimikatz? Does anything related to this processing care whether or not credentials were collected by mimikatz or some other mechanism?